PERFTEST.PS1

```diff
@@ -99,7 +99,8 @@ function test_main {
         [parameter(Mandatory=$true)]  [String] $OutDir = "",
```

Comment:
validate params

Reply:
OutDir does not have to exist as we create Outdir with the UserName due to Linux home directory.